cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/helpers/Util.kt

```diff
@@ -470,6 +472,23 @@ object Util {
         conditionNodes.forEach { n.prevDFGEdges += it }
     }
 
+    /**
+     * Returns the RootPath that belongs to the absolute file path in `file` amongst the
```

Comment:
Please  use  the correct dokka syntax, like `[file]`

Reply:
And the documentation is not accurate, since it will return the parent directory of the file if it does not "belong" to the root path. I would also suggest to use the wording "is relative to" instead of belonging.